### PR TITLE
Add annotations support

### PR DESCRIPTION
**Pull request by atsirin**
**New Features**

- add annotations support

### CODE REVIEW

**Comment by hagen1778** [comment]
Hi @atsirin 

Thank you for contribution!

Looks like `partials/annotations.editor.html` is missing...

**Comment by atsirin** [comment]
Sorry, my mistake. I fixed it

File: src/response_parser.ts [hunk 1]
```diff
@@ -22,4 +24,40 @@ export default class ResponseParser {
...
+
+    for (let i = 0, len = rows.length; i < len; i++) {
+      const row = rows[i];
+
+      result.push({
+        annotation: options.annotation,
+        time: Math.floor(row.time),
+        title: row.title,
+        text: row.text,
+        tags: row.tags ? row.tags.trim().split(/\s*, \s*/) : []
```
Comment (hagen1778) [inline src/response_parser.ts]: There is a space in expression for splitting tags while `Show help` tooltip says `This is a comma separated string of tags e.g. 'tag1,tag2'`. It's misleading

**Comment by atsirin** [comment]
Please see changes. Thank you!

**Comment by hagen1778** [comment]
Merged to master. The new version will be released as soon as I deal with other PRs and issues. Stay tuned and thank you for contribution!

**Pull request merged**